src/Mirza/BusinessRegistry/Handlers/Location.hs

```diff
@@ -77,17 +79,20 @@ addLocationQuery (AuthUser (BT.UserId uId)) locId geoLocId newLoc = do
     guard_ (user_id user ==. val_ uId)
     pure $ user_biz_id user
   case mbizId of
-               -- Since the user has authenticated, this should never happen
+    -- Since the user has authenticated, this should never happen
     Nothing -> throwing _UnexpectedErrorBRE callStack
-    Just bizId -> do
+    Just userBizId -> do
```

Comment:
Yep this looks good, I notice that you've updated the BR test for this which is good and I've got some tests that I've been working on in my as_340_dev branch which should test this out more thoroughly when that branch is merged.